scripts/cxg_admin.py

```diff
@@ -138,6 +150,29 @@ def transfer_collections(ctx, curr_owner, new_owner):
                 exit(0)
 
 
+@cli.command()
+@click.pass_context
+def create_cxg_artifacts(ctx):
+    with db_session_manager() as session:
```

Comment:
Nit: it might be nice to have a print statement that describes the action that is about to be performed.